.github/workflows/check_dependencies.yml

```diff
@@ -0,0 +1,26 @@
+name: Check PR Dependencies
+
+on:
+    pull_request:
```

Comment:
needs to be ```pull_request_target``` without it won't have the permission it needs to add a label.
[Good Explanation](https://securitylab.github.com/research/github-actions-preventing-pwn-requests/)